lib/vsc/zk/rsync/source.py

```diff
@@ -406,8 +408,10 @@ def rsync_path(self, path):
         """ start rsync session for given path and destination, returns true if successful """
         if not path:
             self.log.raiseException('Empty path given!')
-        elif not isinstance(path, basestring):
+            return None
+        elif not is_string(path):
             self.log.raiseException('Invalid path: %s !' % path)
+            return None
```

Comment:
also needed when exception is raised?

Reply:
nvm, can't return too much i guess :stuck_out_tongue:

Reply:
yes, because pylint does not recognise self.log.raiseException as an exception.